src/AppBundle/Form/PostType.php

```diff
@@ -43,13 +43,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'attr' => array('autofocus' => true),
                 'label' => 'label.title',
             ))
-            ->add('summary', 'textarea', array('label' => 'label.summary'))
-            ->add('content', 'textarea', array(
+            ->add('summary', 'Symfony\Component\Form\Extension\Core\Type\TextareaType', array('label' => 'label.summary'))
```

Comment:
I think it doesn't necessary to change it to `text`... Even with `255` length more convenient work with `textarea`.